crates/starknet-devnet/Cargo.toml

```diff
@@ -24,6 +24,7 @@ url = { workspace = true }
 
 # async
 tokio = { workspace = true, features = ["signal"] }
+futures = "0.3"
```

Comment:
Why did you choose specifying the dependency in the crate Cargo.toml instead of the workspace Cargo.toml?

Reply:
7e9121691925303868b0d1fa3b059c242607092c